marker_found: 0
marker_id: 0
marker_locations: [[0.0,0.0],[0.0,0.0],[0.0,0.0],[0.0,0.0]]
broadcast_frame: false
capture: false